modules/launcher/src/test/java/org/mule/module/launcher/coreextension/DefaultMuleCoreExtensionManagerTestCase.java

```diff
@@ -61,20 +70,39 @@ public void injectsDeploymentServiceAwareCoreExtension() throws Exception
     }
 
     @Test
-    public void initializesDeploymentListenerCoreExtension() throws Exception
+    public void initializesApplicationDeploymentListenerCoreExtension() throws Exception
```

Comment:
This are unit tests, but you need to use the new listener on the DeploymentServiceTestCase to test the integration part of it